eox_nelp/edxapp_wrapper/test_backends/event_routing_backends_m_v1.py

```diff
@@ -15,6 +15,7 @@ def get_xapi_constants():
     constants = Mock()
     constants.EN = "en"
```

Comment:
DO we should  to  EN_US constants??
https://github.com/nelc/event-routing-backends/blob/master/event_routing_backends/processors/xapi/constants.py#L93

Reply:
you make me think about a refactor in order to use EN_US instead of default_language, or set default language as constants.EN_US, however I'm not sure if I understood your question here

Reply:
Yes you understooD. May be we have to check the assertion in tests to constants.EN_US.
https://github.com/eduNEXT/eox-nelp/blob/28101dbfe2f9a85b6d2af38f758fa52bf86c5c73/eox_nelp/openedx_filters/tests/xapi/tests_filters.py#L348

Reply:
This requires a refactor since the constants.EN  is in multiple places, that's why that change can not be included in this PR